src/indicators/StateIndicator.js

```diff
@@ -21,6 +21,7 @@ export const StateIndicator = ({
   }, [age]);
 
   const maxNumForGetRandomNumber = 2;
+  const SICK_PET_TIME_TILL_DEATH_MS = 15000;
```

Comment:
better extract such important configuration options into their own file, because you might later need the same variable in a different place